rust/src/task.rs

```diff
@@ -103,3 +109,42 @@ impl Task for RSMem {
         TaskResult(op.into(), start.elapsed())
     }
 }
+
+struct LOCAL {
```

Comment:
all caps means srs business

Reply:
Yessir lol. I'll change to `Local` tho because it looks kinda weird